lib/erlef_web/controllers/blog_controller.ex

```diff
@@ -0,0 +1,42 @@
+defmodule ErlefWeb.BlogController do
+  use ErlefWeb, :controller
+  action_fallback ErlefWeb.FallbackController
+
+  alias Erlef.Blogs
+
+  def index(conn, %{"topic" => topic}) do
+    with {:ok, posts} <- list(topic) do
+      render(conn, "index.html", topic: topic, posts: posts, about: Blogs.Config.about_for(topic))
```

Comment:
Using `with` here feels odd because it's no longer multiple queries, but doesn't hurt anything either.